code/modules/mob/living/living_defense.dm

```diff
@@ -91,12 +91,11 @@
 /mob/living/proc/is_ears_covered()
 	return null
 
-/mob/living/bullet_act(obj/projectile/proj, def_zone, piercing_hit = FALSE)
+/mob/living/bullet_act(obj/projectile/proj, def_zone, piercing_hit = FALSE, blocked = check_projectile_armor(def_zone, proj))
```

Comment:
similar (that or make a wrapper proc for bullet_act that sets this var and also does all the signal stuff)